packages/protocol/blockchain_scripts/migrations/README.md

```diff
@@ -0,0 +1,111 @@
+## Migration documentation
+
+The migration process usually consists of the following steps:
+
+```tsx
+export async function main() {
+  const effects = await getDeployEffects()
+  const {deployedContracts, upgradedContracts} = await deploy(effects)
+  await effects.executeDeferred()
+  return {deployedContracts, upgradedContracts}
+}
+```
+
+There are two types of tests: trivial and mainnet_forking tests. Check this .circleci code (see, `-v` for grep):
+
+```tsx
+- tests
+  tests=$(circleci tests glob "test/**/*.test.ts"| grep -v mainnet_forking | circleci tests split --split-by=timings)
+- mainnet_forking
+  tests=$(circleci tests glob "test/**/*.test.ts"| grep mainnet_forking | circleci tests split --split-by=timings)
+```
+
+If you need to do something specific to the logic
+
+Looks like GF uses this: [Upgrading smart contracts - OpenZeppelin Docs](https://docs.openzeppelin.com/learn/upgrading-smart-contracts)
```

Comment:
`Looks like GF` is from our conversations, I think in documentation it should be more reassuring :)
Also, turned out that not everything from the upgradable-smart-contracts framework is used, only something for proxy contracts. I think this documents needs some more architecture overview, including the usage of gnosis and multistep deployment